update-models.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
```

Comment:
same as the PHP branch, I'd recommend adding something like `set -euo pipefail` so that the script exits on an error

Reply:
Done.